packages/grafana-data/src/datetime/formats.test.ts

```diff
@@ -52,8 +52,8 @@ describe('systemDateFormats', () => {
 
   it('contains correct browser-localized date formats', () => {
     systemDateFormats.useBrowserLocale();
-    expect(systemDateFormats.fullDate).toBe('MM/DD/YYYY, hh:mm:ss A');
-    expect(systemDateFormats.fullDateMS).toBe('MM/DD/YYYY, hh:mm:ss.SSS A');
+    expect(systemDateFormats.fullDate).toBe('MM/DD/YYYY, hh:mm:ss A');
+    expect(systemDateFormats.fullDateMS).toBe('MM/DD/YYYY, hh:mm:ss.SSS A');
```

Comment:
Please see https://github.com/grafana/grafana/issues/74533#issuecomment-1715830938 for explanation of breaking change in node >=18.13.0 and date formats.